src/components/modal/Modal.tsx

```diff
@@ -17,6 +18,7 @@ export default function Modal({
     title,
     content,
     submitText,
+    submitEnabled = true,
```

Comment:
Added prop for enabling/disabling modal submit button (recording submit only enabled when there is a recording).